app/components/modal/FirstModal/FirstModal.module.css

```
@@ -0,0 +1,107 @@
+.firstmodalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
+  font-weight: bold;
+}
+
+.firstmodalContent {
```

[キャメルケース](https://developer.mozilla.org/ja/docs/Glossary/Camel_case)に統一でお願いします！
```suggestion
.firstModalContent {
```